pkg/sink/sink_test.go

```diff
@@ -1140,7 +1141,13 @@ func (f *echoInterceptor) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	defer r.Body.Close()
+	if f.counter == 0 {
```

Comment:
We are using the same interceptor twice as if there were two different `Webhooks`.

The first webhook adds a new key to the extensions key called `truncated_sha` that was added by first `cel` interceptor.

The second webhook adds a new key to the root extensions key.